quad_control/setup.py

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
You have to explain to me the purpose of this setup, and when to include stuff in this setup